src/routes/resource-edit.js

```diff
@@ -77,24 +77,24 @@ class ResourceEditRoute extends Component {
     if (values.isSelected === false && model.package.isCustom) {
       destroyResource(model);
     } else if (values.isSelected === false) {
-      model.isSelected = false;
-      model.customCoverages = [];
-      model.visibilityData.isHidden = false;
-      model.identifiersList = [];
-      model.identifiers = [];
-      model.customStatement = '';
-      model.customEmbargoPeriod = {};
-      model.contributors = [];
-      model.coverageStatement = '';
-      model.proxy = {};
-
-      updateResource(model);
+      updateResource(Object.assign(model, {
```

Comment:
I believe it is the same as it was before as we will mutate the model.
You can pass the first argument (target) of object.assign as an empty object and then it will not be mutation